torch/_dynamo/config.py

```diff
@@ -69,9 +69,8 @@
 # legacy config, does nothing now!
 dynamic_shapes = True
 
-use_lazy_graph_module = (
-    os.environ.get("TORCH_COMPILE_USE_LAZY_GRAPH_MODULE", "1") == "1"
-)
+# legacy config, does nothing now!
+use_lazy_graph_module = True
```

Comment:
We still use this flag here - https://github.com/pytorch/pytorch/blob/9bc9d4cdb4355a385a7d7959f07d04d1648d6904/torch/_inductor/compile_fx.py#L576

You will have to remove that line as well. Please do one more search throughout the repo (I did a grep and found this one) and remove the usages of this flag.